lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -105,6 +126,14 @@ def test_empty_topic_id(self):
         self.form_data.setlist("topic_id", ["", "not empty"])
         self.assert_error("topic_id", "This field cannot be empty.")
 
+    @ddt.data(*itertools.combinations(["topic_id", "text_search"], 2))
```

Comment:
interesting, i haven't seen `combinations` before, but what does it add here?   it seems to be a no-op
```
>>> list(itertools.combinations(["a", "b"], 2))
[('a', 'b')]
```